content/en/security/application_security/troubleshooting.md

```diff
@@ -564,28 +564,41 @@ See the Application Security [set up documentation][11] to validate you you are
 
 Ensure the `DD_INSTRUMENTATION_TELEMETRY_ENABLED` environment variable (`DD_TRACE_TELEMETRY_ENABLED` for NodeJS) is set to `true`, or the corresponding system property for your language is enabled. For example in Java: `-Ddd.instrumentation.telemetry.enabled=true`
 
-## Disabling Threat Detection and Protection
+## Disabling threat detection and protection
 
 To disable ASM, remove the `DD_APPSEC_ENABLED=true` environment variable from your application configuration, and restart your service.
```

Comment:
ASM is the portfolio of products including SCA, Code security, & Threat management. In this case, this section is specific to threat management. We should avoid using the larger ASM term.

Additionally, we should review our use of the acronym ASM in the docs and replace with the appropriate product name when clarification is needed - (Not sure if in scope of this PR, so feel free to disregard).